packages/block-library/src/post-title/edit.js

```diff
@@ -60,10 +58,10 @@ export default function PostTitleEdit( {
 					value={ rawTitle }
 					onChange={ setTitle }
 					__experimentalVersion={ 2 }
-					{ ...( isLink ? {} : blockProps ) }
```

Comment:
This element is never rendered when `isLink` is true, so the check is redundant.